hackathon_site/dashboard/frontend/src/components/general/TitledPaper/TitledPaper.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import TitledPaper from "./TitledPaper";
+
+describe("<TitledPaper />", () => {
+    const titleTest = "Test title";
+    const childTest = "Test child";
+    it("matches snapshot", () => {
+        const { asFragment } = render(
+            <TitledPaper title={titleTest}>{childTest}</TitledPaper>
+        );
+        expect(asFragment()).toMatchSnapshot();
+    });
+    it("renders text given to it", () => {
+        const { getByText } = render(
+            <TitledPaper title={titleTest}>{childTest}</TitledPaper>
+        );
+        expect(getByText(titleTest)).toBeInTheDocument();
+        expect(getByText(childTest)).toBeInTheDocument();
+    });
+    it("renders no text if not given anything", () => {
```

Comment:
This is a strange test. There's no possible way that `titleTest` and `childTest` could even get into the component, so this test can't possibly fail. 

Perhaps a useful test in it's place would be that the `Typography` component isn't rendered if no title is given (you'd probably need to give it a `data-testid`).